app/javascript/src/components/Dashboard/Contacts/Avatar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Avatar as NeetoAvatar, Typography } from "neetoui";
+
+const Avatar = ({ user }) => {
+  return (
+    <div className="flex items-center space-x-2">
+      <NeetoAvatar size="large" user={user} />
+      <div>
```

Comment:
Why do we need to make this a div?

Reply:
The first div has class flex and second div is inside the first div and its purpose is to group all the typography one after another (class block).

Reply:
@vijay2136 https://www.loom.com/share/cb2050ec9f9748909251f81d78802912

Reply:
https://vimeo.com/668636745/6cd40ea1f0

Reply:
Got it.